### DISCUSSION TITLE
How to find Plate Type ID

### DISCUSSION THREAD

**Discussion by DanRothwell** [ukrobotics/DispenseLibDotNet#18] [Q&A] [answered]
When running a protocol from DispenseLib, we need to provide a plate type GUID. How can we find the GUID of a plate type? The only information I can find is name, part number and well count.
![image](https://github.com/ukrobotics/DispenseLibDotNet/assets/111614151/dbc6c414-52c4-4000-9966-b2806e8a59ee)

**Comment by mcounsell** [comment, marked as answer]
We have a public plate library that the D2 codebase is using.  

See here:
https://labware.ukrobotics.app/

Here you will find the GUIDs for each plate type. 

We should probably show these plate type IDs on the D2 web app... I'll create a task for this but for now... please see the URL above and let me know if this works for you @DanRothwell
